### ISSUE TITLE
Distinguish between active and sleeper agents

### ISSUE THREAD

**Issue by dnesting** [paultag/mandelbrot#40]
Some people have left USDS to return to their careers, but are still on contract.  These should be visually distinct.

(Also, what's the intended behavior for people whose contracts have expired?  Carla and Nathan are not on the list, but that may be expected.)

**Comment by KHarshawat** [comment]
Both good points. Here are some options
- Maybe we use icons in the top right corner of the card (with a tooltip) to differentiate. For example an hour-glass might be for people who are still on contract but not actively working on anything and a frown for someone who is gone: 
  ![image](https://cloud.githubusercontent.com/assets/6335546/12936295/79587208-cf6b-11e5-8de3-6b27d40a71ff.png) 
  ![image](https://cloud.githubusercontent.com/assets/6335546/12936369/24f242ce-cf6c-11e5-855a-edeca795f90e.png)
- We also use thumbs up /down for active and inactive so maybe re-using the same icons would be good.  
- It might be better to just stick with a simple asterisk. 
- Alternately we could change the color of the cards themselves (and potentially the background of their profile page). So a person who is no longer active has a card that is grey. This probably doesn't work as well as the icons though.
- We could also add a colored bottom border to the card, though you run into the same issue with the profile page.

**Comment by paultag** [comment]
Yeah, good points. I'm going to have to change active to an enum.
